hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/StreamSourceStageTestBase.java

```diff
@@ -103,6 +103,7 @@ protected void test(
             return;
         }
         stageWithTimestamps
+                .peek()
```

Comment:
leftover ?

Reply:
There are just 2 items in the source logged, this helps debugging.